north/src/runtime/linux/cgroups.rs

```diff
@@ -12,7 +12,8 @@
 //   See the License for the specific language governing permissions and
 //   limitations under the License.
 
-use crate::{Event, EventTx, SETTINGS};
+use super::super::super::runtime::{config, Event, EventTx};
```

Comment:
maybe nicer:
```
use crate::runtime::{config, Event, EventTx};
```

Reply:
The idea is to have inner module `use` directive relative and only needed ones absolute based on the crate root.

Reply:
ok...looks weird but you are the expert